modules/ROOT/pages/managing-your-account.adoc

```diff
@@ -129,6 +129,17 @@ If your organization uses xref:access-management::external-identity.adoc[Single
 . Click *Continue*.
 You are redirected to your identity provider.
 
+== Allowlist URLs to Log in to and Use Anypoint Platform
+
+If you are using a restricted environment such as a virtual machine or have proxy requirements that prevent you from logging in to Anypoint Platform, add the following URLs to your allowlists so your sessions are not blocked or interrupted by your organization's firewall or proxy:
```

Comment:
```suggestion
If you are using a restricted environment, such as a virtual machine, or have proxy requirements that prevent you from logging in to Anypoint Platform, add the following URLs to your allowlists so your sessions are not blocked or interrupted by your organization's firewall or proxy:
```

Reply:
This sentence is pretty long and hard to process. It might be better to split it into two sentences for readability.

Reply:
Do you have a recommendation here? I merged without seeing this but will open a new PR to fix.